requirements.txt

```diff
@@ -22,3 +22,4 @@ umap-learn>=0.3.10,<0.5
 legacy-api-wrap
 packaging
 sinfo
+openTSNE>=0.5.0
```

Comment:
This needs to be updated to 0.6, when I release it. Until then, CI will fail here.